recipes/aioshutil/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "aioshutil" %}
+{% set version = "1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aioshutil-{{ version }}.tar.gz
+  sha256: ddabe1748c8a71ec3c7d213a4d0cf58fb495c71419334a0575b1f8a3be8373e8
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
```

Since this is pure Python, it would be much better to make this a noarch build:
```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
```